src/__tests__/components/followerTest/followerViewContainer.test.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from '../../../../config/enzymeConfig';
+import AllUserView, { FollowerView as View } from '../../../containers/followerViewContainer';
+import { mockStore, initialState } from '../../../__mocks__/store';
+import {} from '../../../components/Header/Header';
+import user from '../../../__mocks__/user';
+
+document.body.innerHTML =
+'<div id="myModal"></div> ' +
+'<input id="search-btn" value="h"/>' +
+'<h5 class="usernamevalue">khd</h5>' +
+'<div class="cards"></div> ';
+
+jest.mock('../../../components/Header/Header', () => () => (
+    <div id="mocked">hello</div>
+))
+
+describe('<following />', () => {
+    test('should test ', () => {
+        const component = mount(<Provider 
+            store={mockStore({
+              ...initialState,
+              user: { ...initialState.user, profile: { ...user }, userProfile: { ...user } },
+              followerData:{
+                  getAllUsers:{data:{}},
+                  getAllfollower:{data:{},
+                  
+                },
+                follow:{message:''},
+                getAllfollowing:{data:{following:[] 
+            }},
+                unfollow:{message:''}
+                },
+              getDataThunkPrivate: jest.fn(), 
+              signupSuccess: { isAuth: true },
+              userCredentials: { profile: { image: '' } },
+              follower: 0,
+              following: 0,
+              followMessage: ''
+            })}
+          >
+            <MemoryRouter>
+              <AllUserView />
+            </MemoryRouter>
+          </Provider>);
+
+        expect(component).toBeDefined();
+    });
+
+    test('should test view profile event follow event', () => {
+
+        const props = {
+            follower: [
+                    {follower:{
+                  username: 'kagabo',
+                  image: '',
+                  username: '',}
+                  },{follower:{
+                  username: 'ewgehe',
+                  image: '',
+                  username: '',}
+                  },{follower:{
+                  username: 'wergkjw',
+                  image: '',
+                  username: '',}
+                      }
+                  ],
+            followMessage: 'test'
+        };
+        const component = mount(
+            <MemoryRouter>
+              <View { ...props } />
+            </MemoryRouter>);
+          component.setProps({followMessage: 'dwkjhf'})
+          // component.find("button[id='0followbtn']").simulate('click', {  });
```

Comment:
Great work on this PR @kagabof, Please remove commented code if you are not using it.